CMakeLists.txt

```diff
@@ -38,3 +38,27 @@ include(build_cmake/configs/${WT_ARCH}/${WT_OS}/config.cmake)
 include(build_cmake/configs/auto.cmake)
 # Load WiredTiger related configuration options
 include(build_cmake/configs/base.cmake)
+
+set(link_type "STATIC")
+if(NOT ENABLE_STATIC)
+    set(link_type "SHARED")
```

Comment:
This makes me think of 'STATIC' being the default link type (though looks not) 🙂 Just a nit, feel free to ignore.

Reply:
True, this doesn't read clearly given the default config is a shared library build. 
I've updated the statement, restructuring the logic to make it more clearer :)